wide-github-toggle.js

```diff
@@ -1,16 +1,13 @@
 function setDisabledClass(obj) {
-  if (obj.enableWideGithub)
-  {
+  if (obj.enableWideGithub) {
     document.body.classList.remove('wgh-disabled');
-  }
-  else
-  {
+  } else {
     document.body.classList.add('wgh-disabled');
   }
 }
 
-chrome.runtime.onMessage.addListener(function(request, sender, callback) {
+chrome.runtime.onMessage.addListener((request, sender, callback) => {
```

Comment:
While technically "okay" you might want to confirm that this is wanted to have newer syntax.